sparkles/tests/test_checks.py

```diff
@@ -896,7 +896,7 @@ def test_check_guide_geometry(aca_review_table):
     assert len(acar.messages) == 1
     msg = acar.messages[0]
     assert msg["category"] == "critical"
-    assert 'Guide indexes [4, 5, 6] clustered within 500" radius' in msg["text"]
+    assert 'Guide indexes [4, 7, 8] clustered within 500" radius' in msg["text"]
```

Comment:
I was not immediately anticipating that *guide*-related test would change, but these guide indexes are the overall catalog idx values, and acquisition-related changes fiddle with order of BOT stars in the list.  I confirmed that these are the same 3 stars.